synapseformation/utils.py

```diff
@@ -0,0 +1,51 @@
+import json
+
+import yaml
+
+
+def read_yaml_config(template_path: str) -> dict:
+    """Get yaml synapse formation template
+
+    Args:
+        template_path: Path to yaml synapse formation template
+
+    Returns:
+        dict for synapse configuration
+    """
+    with open(template_path, "r") as template_f:
+        template = yaml.load(template_f, Loader=yaml.FullLoader)
```

Comment:
Do we have a reason not to use `safe_load()`? Source: https://stackoverflow.com/a/1774043

Reply:
Thanks for mentioning this, theres no particular reason why I'm doing this other than my legacy code in other projects.